x-pack/legacy/plugins/siem/public/pages/case/components/configure_cases/closure_options.test.tsx

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import { ClosureOptions } from './closure_options';
+import { useMountAppended } from '../../../../utils/use_mount_appended';
+import { TestProviders } from '../../../../mock';
+import { ClosureOptionsRadio } from './closure_options_radio';
+
+describe('ClosureOptions', () => {
+  const mount = useMountAppended();
+
+  test('it renders', () => {
+    const wrapper = shallow(
```

Comment:
I think we could use `mount` to verify all the cases, and move the renderer into [beforeAll](https://jestjs.io/docs/en/api#beforeallfn-timeout), so if some props updated in the future, we don't have to change it every where